packages/common-types/src/utils.ts

```diff
@@ -13,9 +13,21 @@ import { EncryptedMessage } from "./baseTypes/commonTypes";
 // const privKeyBnToPubKeyECC = (bnPrivKey) => {
 //   return getPublic(privKeyBnToEcc(bnPrivKey));
 // };
+export type KeyType = "secp256k1" | "ed25519";
 
 export const ecCurve = new EC("secp256k1");
 
+const ellipticSecp256k1 = new EC("secp256k1");
+const ellipticEd25519 = new EC("ed25519");
+
+export function getEllipticCurve(keyType: KeyType = "secp256k1") {
```

Comment:
we could just use `new EC()`
do we need to define `ecCurve` and `getEllipticCurve`

Reply:
will be easy in case we support more curves, we can change in single place?

Reply:
hmm..

getEllipticCurve("ed25519")
vs new EC("ed25519")